docs/source/notebooks/wps_resolve_rules_demo.ipynb

```diff
@@ -153,6 +153,55 @@
     "    )"
```

Comment:
<p>Perhaps you could add some comments (or markdown cells) to describe the different ways we are collecting outputs here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pacificclimate/sandpiper/pull/51/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>